floodwatch/src/js/components/Compare.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {FilterParent} from './FilterParent';
+import AdBreakdown from '../../stubbed_data/adbreakdown_response.json';
+import Filters from '../../stubbed_data/filter_response.json';
+
+// import '../../Compare.css';
+
+type CompareState = {}
```

Comment:
This type means that `CompareState` is literally always an empty object, but it doesn't appear to be getting set as such in the constructor.